packages/desktop-client/src/components/reports/spreadsheets/summary-spreadsheet.ts

```diff
@@ -196,13 +199,19 @@ function calculatePerMonth(
     amount: monthlyData[d.format(m, 'yyyy-MM')] || 0,
   }));
 
+  const dayOfMonth = months.at(-1)!.getDate();
```

Comment:
minor suggestion: `months.at(-1)!` is repeated twice here and below, maybe we could make it a bit more clean by extracting a variable called `lastMonth` so that we don't have to repeat the `!` assertion

Reply:
Done :)